src/Websites/Platform/src/Bit.Websites.Platform.Client/Pages/Templates/Templates04CreateProjectPage.razor.scss

```
@@ -29,7 +29,13 @@
 
 .image {
     @include Image;
+    // The Image mixin's fixed 85% height distorts these diagrams inside the 350px-tall
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)